models/generators.py

```diff
@@ -33,15 +35,18 @@ def __init__(self, img_size, num_latent_dims, kernel_size, num_filters,
         self.reshape_layer = Reshape(reshape_dims)
         self.upsample1 = UpSampling2D()
         self.conv1 = Conv2D(filters=num_filters, kernel_size=kernel_size, activation='relu', padding='same')
+        self.bn1 = BatchNormalization()
         self.upsample2 = UpSampling2D()
         self.conv2 = Conv2D(filters=num_output_channels, kernel_size=kernel_size, activation='tanh', padding='same')
 
     @tf.function
-    def call(self, noise):
-        x = self.dense1(noise)
+    def call(self, noise, embedding):
```

Comment:
@alecokas can I assume embedding will be passed in here?

Reply:
Yeah I think that makes sense